examples/with-claims/api/claims/EmailVerifiedClaim.ts

```diff
@@ -0,0 +1,30 @@
+import { SessionClaimValidator } from "supertokens-node/recipe/session";
+import { BooleanClaim } from "supertokens-node/recipe/session/claims";
+import { isEmailVerified } from "supertokens-node/recipe/emailpassword";
+
+class EmailVerifiedClaimClass extends BooleanClaim {
```

Comment:
We need to make this example app much simpler to consume and understand:
- We will remove this claim since it will be prebuilt
- Rename the claims folder to `customClaims`
- RolesClaim should also be removed since it should be inbuilt.
- The custom APIs that use claims should be fewer in number - for example, we do not need to show refreshing or email verification claim API.

Keeping this comment open until everything else is done, and we can work on cleaning up the example app as the last step.